pontoon/checks/libraries/pontoon.py

```diff
@@ -65,7 +66,7 @@ def run_checks(entity, string):
                 )
 
     # Prevent empty translation submissions if not supported
-    if resource_ext not in {'properties', 'ini', 'dtd', 'inc'} and string == '':
+    if resource_ext not in Resource.EMPTY_TRANSLATION_FORMATS and string == '':
```

Comment:
[nit] Do you think that property here could be a good idea?
e.g. `if entity.resource.allow_empty_translation and string == ''`?

Reply:
Yes. :) I'll add that.